proposed/http-client/http-client.md

```diff
@@ -41,12 +41,11 @@ use Psr\Http\Message\ResponseInterface;
 interface ClientInterface
 {
     /**
-     * Sends a PSR-7 request.
-     *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
-     *
-     * If a request is sent without any prior configuration, a HTTP client MUST NOT follow redirects.
+     * Sends a PSR-7 request and returns a PSR-7 response. 
+     * 
+     * Every technically correct HTTP response MUST be returned as is, even if it represents a HTTP 
+     * error response or a redirect instruction. The only exception is 100 request which MUST be 
```

Comment:
The only exception is 1xx responses which MUST be assembled in the HTTP client.

Reply:
Thank you, I've updated the PR